go/runtime/client/watcher.go

```diff
@@ -216,17 +238,13 @@ func (w *blockWatcher) Stop() {
 	close(w.stopCh)
 }
 
-func newWatcher(common *clientCommon, id common.Namespace) (*blockWatcher, error) {
+func newWatcher(common *clientCommon, id common.Namespace, p2pSvc *p2p.P2P) (*blockWatcher, error) {
 	committeeWatcher, err := committee.NewWatcher(
```

Comment:
Why is the committee watcher (and committee client) even used now (since tx dispatch happens over p2p anyway)?

Reply:
probably not needed, will remove